ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/EthFeeHistory.java

```diff
@@ -279,19 +283,9 @@ private List<Wei> boundRewards(final List<Wei> rewards) {
    * @return The bounded reward.
    */
   private Wei boundReward(final Wei reward, final Wei lowerBound, final Wei upperBound) {
-
-    // If the reward is less than the lower bound, return the lower bound.
-    if (reward.compareTo(lowerBound) <= 0) {
-      return lowerBound;
-    }
-
-    // If the reward is greater than the upper bound, return the upper bound.
-    if (reward.compareTo(upperBound) > 0) {
-      return upperBound;
-    }
-
-    // If the reward is within the bounds, return the reward as is.
-    return reward;
+    return reward.compareTo(lowerBound) <= 0
+        ? lowerBound
+        : reward.compareTo(upperBound) > 0 ? upperBound : reward;
```

Comment:
```suggestion
        : (reward.compareTo(upperBound) > 0) ? upperBound : reward;
```

Reply:
nit: this same logic in EthGasPrice has an extra set of () - idk which is easier to read but they should match

Reply:
another nit is that line 286 uses <= in the comparison but 288 uses strictly > but >= would have the same effect and would be more symmetrical

Reply:
done